src/wolfkrow/core/tasks/task.py

```diff
@@ -694,7 +694,7 @@ def from_dict(
         resolver = Resolver(replacements, resolver_search_paths, sgtk=sgtk)
 
         if replacements:
-            resolver.resolve(data_dict)
+            data_dict = resolver.resolve(data_dict)
```

Comment:
Is this just to clean up the code or fixing an actual bug? "data_dict" here is always a dictionary, and so we never actually need to re-assign the value because the resolver modifies the value passed in directly.

Though that is bad practice... and this is better. I just want to make sure there isn't some other edge case coming through or something which caused a bug here.